torch/nn/functional.py

```diff
@@ -1655,25 +1657,43 @@ def pixel_shuffle(input, upscale_factor):
         upscale_factor (int): factor to increase spatial resolution by
 
     Examples::
+        # 1D example
+        >>> ps = nn.PixelShuffle(2)
+        >>> input = autograd.Variable(torch.Tensor(1, 4, 8))
```

Comment:
lets remove Variable wrappers in example code

Reply:
I think pixel_shuffle 1D is not what we want
<pre>x = torch.arange(8).view(1, 2, 4)
print(x)
print(x.shape)
y = pixel_shuffle(x, 2)
print(y.shape)
print(y)
sys.exit()<pre>
the output is tensor([[[0, 1, 2, 3, 4, 5, 6, 7]]])
but we expect [0, 4, 1, 5, 2, 6, 3, 7]
pixel_shuffle 2d is right

Reply:
As you can see, pixel_shuffle 1D  just do like torch.cat(),  not pixel_shuffle
@SsnL